### PR TITLE
feat(blog): add post on optimizing VSCode Workflow/Setup

### PR DESCRIPTION
**Pull request by sikatikenmogne**
## Description
This pull request adds a new blog post titled "Advanced VSCode Customization: **How to Optimize Your VSCode Workflow/Setup**". The post covers advanced customization techniques for Visual Studio Code (VSCode) to help developers optimize their development environment. It includes recommendations for essential extensions, UI customization, settings configuration, tool integration, and task automation.

## Changes
- Added a new markdown file [`how-to-optimize-your-vscode-workflow-setup.mdx` ](https://github.com/sikatikenmogne/oss-cameroon-blog/blob/blog-post/advanced-vscode-customization/data/blog/how-to-optimize-your-vscode-workflow-setup.mdx)with the blog post content.
- Included tags for better categorization and SEO.

## Checklist
- [x] The post has been tested for proper formatting in the blog layout.
- [x]  All links and references have been verified.
- [x] The blog post content has been proofread and reviewed.

## Preview

image

Download PDF -> [How to optimize your VSCode Workflow_Setup _.pdf](https://github.com/user-attachments/files/17017075/How.to.optimize.your.VSCode.Workflow_Setup._.pdf)

## Tested environments
- Github Codespaces
- Windows 11

## Related Pull Request
- You should merge #35 before to avoid any issues with author references

  e.g: 
  > Error: ENOENT: no such file or directory, open 'C:\Users\Windows\Documents\GitHub\sikatikenmogne\oss-cameroon-blog\data\authors\sikatikenmogne.md'

  For testing purpose you should cherry pick the commit [chore(authors): add Samuel SIKATI as author](https://github.com/osscameroon/blog/pull/35/commits/cca5e3aaa06f827283f086f18c52388f3a28e040) as follow:

  ```sh
  git cherry-pick --no-commit cca5e3aa
  ```

## Additional Notes
- Feedback and suggestions for improvement are welcome.

### CODE REVIEW

**Comment by pythonbrad** [comment]
Hi @sikatikenmogne, I think it's better to separe "adding sikatikemogne as author" and "add post on optimizing VSCode Workflow/Setup"

**Comment by sikatikenmogne** [comment]
Okay I did it, what do you think about my blog post subject and content btw.

**Review by RMPR — COMMENTED** [review]
Very nice work I like your willingness to document everything. I have one comment. Can you squash the commits into one (and force push) to make it easier to review the actual blogpost?

**Comment by sikatikenmogne** [comment]
> Very nice work I like your willingness to document everything. I have one comment. Can you squash the commits into one (and force push) to make it easier to review the actual blogpost?

Thank you, I did it and I a have already setup a codespace for the reviewing so you can consult my blog post by the following link :
https://supreme-spork-p95p65r5vgrh66xx-3000.app.github.dev/posts/how-to-optimize-your-vscode-workflow-setup

File: data/blog/how-to-optimize-your-vscode-workflow-setup.mdx [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+#### Importing Keybindings from Other IDEs
+
+VSCode supports importing keybindings from other popular IDEs, allowing you to use familiar
+shortcuts. Here’s how you can import keybindings:
+
+1. Open the Extensions view by pressing `Ctrl+Shift+X`.
+2. In the search bar, type the name of the IDE you want to import keybindings from (e.g.,
+   `Visual Studio Keymap`, `Sublime Text Keymap`, `Atom Keymap`).
+3. Click on the keymap extension you want to install and then click the `Install` button.
+4. After installation, the keybindings from the selected IDE will be applied to VSCode.
```
Comment (RMPR) [inline data/blog/how-to-optimize-your-vscode-workflow-setup.mdx:536]: Does this require restarting vs code?

**Review by RMPR — COMMENTED** [review]
Nice work only one minor comment/question.

File: data/blog/how-to-optimize-your-vscode-workflow-setup.mdx (hunk 1 above)
Comment (sikatikenmogne) [reply data/blog/how-to-optimize-your-vscode-workflow-setup.mdx:536]: No, importing keybindings from other IDEs into VSCode does not typically require restarting VSCode. Once you install the keymap extension, the new keybindings should be applied immediately.

**Review by RMPR — APPROVED** [review]
LGTM, one last comment, can you get rid of the merge commit?

**Comment by sikatikenmogne** [comment]
> LGTM, one last comment, can you get rid of the merge commit?

Sorry 😅 , done

**Comment by RMPR** [comment]
Thank you very much for your contribution. Well appreciated.

**Pull request merged**